cmd/cli/kubectl-kyverno/commands/create/permission/command.go

```diff
@@ -0,0 +1,92 @@
+package permission
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"text/template"
+
+	"github.com/Masterminds/sprig/v3"
+	"github.com/kyverno/kyverno/cmd/cli/kubectl-kyverno/commands/create/templates"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	Verbs        []string
+	Controllers  []string
+	ApiGroup     string
+	ResourceType string
+}
+
+func Command() *cobra.Command {
+	var verbs []string
+	var path string
+	var opts options
+	cmd := &cobra.Command{
+		Use:   "permission [resource-type]",
+		Short: "Create an aggregated role for a given resource type",
+		Long: `This command generates a Kubernetes ClusterRole for a specified resource type.
+The output is printed to stdout by default or saved to a specified file.
+Required flags include 'api-group' and 'verbs'.`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			resourceType := args[0]
```

Comment:
Instead of taking a argument, can we  make `--resources` a required flag and allow the user to specify a list of resources?